### ISSUE TITLE
Remote Support does not work in the latest versions of Nvda Alfa 

### ISSUE THREAD

**Issue by Nikita34196** [NVDARemote/NVDARemote#180]
Hello. In the latest versions of NVDA, the add-on does not work when enabled NVDA gives an error and everything.
NVDA version alpha-15968,10635ac2
log.
INFO - main (10:09:33.137):
Starting NVDA
INFO - core.main (10:09:33.247):
Config dir: C:\Users\nikit\AppData\Roaming\nvda
INFO - config.ConfigManager.loadConfig (10:09:33.247):
Loading config: C:\Users\nikit\AppData\Roaming\nvda\nvda.ini
INFO - core.main (10:09:33.496):
NVDA version alpha-15968,10635ac2
INFO - core.main (10:09:33.496):
Using Windows version 10.0.17751 workstation
INFO - core.main (10:09:33.496):
Using Python version 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:22:17) [MSC v.1500 32 bit (Intel)]
INFO - core.main (10:09:33.496):
Using comtypes version 1.1.3
INFO - core.main (10:09:33.496):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - external:synthDrivers.RHVoice.SynthDriver.init (10:09:33.709):
Using RHVoice version 0.7.0
INFO - synthDriverHandler.setSynth (10:09:33.714):
Loaded synthDriver RHVoice
INFO - core.main (10:09:33.714):
Using wx version 4.0.3 msw (phoenix) wxWidgets 3.0.5
INFO - brailleInput.initialize (10:09:33.721):
Braille input initialized
INFO - braille.initialize (10:09:33.723):
Using liblouis version 3.7.0
INFO - braille.BrailleHandler.setDisplayByName (10:09:33.723):
Loaded braille display driver noBraille, current display has 0 cells.
WARNING - core.main (10:09:33.792):
Java Access Bridge not available
INFO - UIAHandler.UIAHandler.MTAThreadFunc (10:09:33.802):
UIAutomation: IUIAutomation6
ERROR - globalPluginHandler.initialize (10:09:34.621):
Error initializing global plugin <class 'globalPlugins.remoteClient.GlobalPlugin'>
Traceback (most recent call last):
File "globalPluginHandler.pyc", line 32, in initialize
File "C:\Users\nikit\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient_init.py", line 56, in init
File "C:\Users\nikit\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient_init.py", line 104, in create_menu
AttributeError: 'MenuItem' object has no attribute 'SetCheckable'
INFO - core.main (10:09:34.759):
NVDA initialized

**Comment by leonardder** [comment]
Duplicate of #176
